### ISSUE TITLE
Small change to inject≤

### ISSUE THREAD

**Issue by xekoukou** [agda/agda-stdlib#675]
```agda
inject≤ : ∀ {m n} → Fin m → m ℕ.≤ n → Fin n
inject≤ {n = zero} zero ()
inject≤ {n = suc n} zero le = zero
inject≤ (suc i) (s≤s le) = suc (inject≤ i le)
```
https://agda.github.io/agda-stdlib/Data.Fin.Base.html#3219

This way, we avoid to need to pattern match on ```le``` , which can be hard to do on a proof, we would need to abstact over ```le``` so as to pattern match.

**Comment by gallais** [comment]
You could also use [`≤-pred`](https://agda.github.io/agda-stdlib/Data.Nat.Properties.html#2803) in the `suc i` case.

**Comment by xekoukou** [comment]
Yes, it is much better now : 
```agda
inject≤ : ∀ {m n} → Fin m → m ℕ.≤ n → Fin n
inject≤ {n = zero} zero ()
inject≤ {n = suc n} zero le = zero
inject≤ {n = zero} (suc i) ()
inject≤ {n = suc n} (suc i) le = suc (inject≤ i (≤-pred le))

```

**Comment by MatthewDaggitt** [comment]
Sounds good.

**Comment by gallais** [comment]
Note that with 2.6.0 we should be able to leave out the impossible cases.

**Comment by xekoukou** [comment]
The same can be done to ```fromℕ≤ ``` . 
I can upload the changes now, or later after 2.6.0

**Comment by MatthewDaggitt** [comment]
Probably best to wait until after 2.6.0 and I reset the CHANGELOG

**Comment by andreasabel** [comment]
I think this should wait until 2.6.0.1 is out, see https://github.com/agda/agda/issues/3692. ;)

**Comment by MatthewDaggitt** [comment]
Fixed with 1690f69
